pkgs/by-name/ce/ceph/rocksdb_7_9/default.nix

```diff
@@ -0,0 +1,140 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  ninja,
+  bzip2,
+  lz4,
+  snappy,
+  zlib,
+  zstd,
+  windows,
+  enableJemalloc ? false,
+  jemalloc,
+  enableLiburing ? stdenv.hostPlatform.isLinux,
+  enableLite ? false,
+  enableShared ? !stdenv.hostPlatform.isStatic,
+  sse42Support ? stdenv.hostPlatform.sse4_2Support,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "rocksdb";
+  version = "7.9.2";
+
+  src = fetchFromGitHub {
+    owner = "ceph"; # note this is a fork, not the upstream `facebook`
+    repo = finalAttrs.pname;
+    # Version pinned by Ceph `src/rocksdb` submodule, last update from:
+    #     https://github.com/ceph/ceph/tree/v19.2.3/src
+    rev = "9fa4990159853479a222244574ca41202e4c95c1";
+    sha256 = "sha256-g0CCiWHG/jjn/fumHGXp88PjOGuRjrKC3ETCy6ffWvQ=";
```

Comment:
Hmm I'm getting a mismatch on this hash when building:

```
         specified: sha256-g0CCiWHG/jjn/fumHGXp88PjOGuRjrKC3ETCy6ffWvQ=
            got:    sha256-JJXtN8mk8fRsGzyimw4e7tN2/c1RJ4Ei6YioVIMhvl0=
```

Reply:
Hm, very weird. I can repro what you say. I'll push a fix but am not sure how it built yesterday given this.